### ISSUE TITLE
Can't connect to Paul Neuhaus Q Series

### ISSUE THREAD

**Issue by mrstegeman** [WebThingsIO/zigbee-adapter#24]
Originally from: https://github.com/mozilla-iot/gateway/issues/921

> Hi,
> 
> I tried to connect to a color temp adjustable Paul Neuheus lamp (Q Series). The remote control will be recognized, but the lamp itself not.
> 
> I disconected all existing learned connections and started the "New Device" search withing the gateway. I hold the RasPi close to the lamps antenna as described in the lamps manual.
> 
> I updated to 0.4.0 but the ZigBee AddOn update doesn't work.
> 
> Any hints you can give me? Anything I can do from the console?
> 
> Thanks,
> Vija

**Comment by dhylands** [comment]
Did you factory reset the light? If it was paired with the remote then it won't pair with the gateway until its put into an unpaired state (and doing a factory reset typically puts it into an unpaired state).

**Comment by Vijabei** [comment]
Yes I did. The remote control and the lamp itself has to be resetted before any new connections are possible.

I will try again to pair them because the error with the not updatable zigbee addon disappeared.

It should work out of the box and I don't know why it doesn't.

**Comment by Vijabei** [comment]
Ok. I tried with 4.1 again. Actually the lamp can be linked and the lamp confirm that with blinking. It is recognised as dimmable lamp, although it is a dimmable color temperature lamp. Also there is no function when hitting the on/off button or trying to use the dimmable feature.
